src/SFA.DAS.AssessorService.Database/Script.PostDeployment1.sql

```diff
@@ -90,5 +90,17 @@ END
 
 -- ON-2242
 :r .\Update-Staff-Reports-Config.sql
-
 -- END
+
+-- ON-2088
```

Comment:
You need to create your sequence as part of the database project - see Apply for an example otherwise a subsequent deployment may remove it (if delete sync was turned on)

Reply:
Ok

Reply:
Done